state/subnets.go

```diff
@@ -361,7 +362,7 @@ func (st *State) addSubnetOps(id string, args network.SubnetInfo) (subnetDoc, []
 	if !unique {
 		return subnetDoc{}, nil, errors.AlreadyExistsf("subnet %q", args.CIDR)
 	}
-	if args.SpaceID == "" && args.SpaceName == network.DefaultSpaceName {
+	if args.SpaceID == "" && args.SpaceName == "" {
```

Comment:
`args.SpaceName` is not used by anything in this method, which leaves open the _logical_ possibility of having an empty space ID and a non-empty space name, meaning we end up with "" instead of `network.DefaultSpaceId` written.

Reply:
removed check on empty string.